front/pages/w/[wId]/ds/index.tsx

```diff
@@ -168,6 +321,110 @@ export default function DataSourcesView({
           </div>
         </div>
       </div>
+      <div className="mx-auto space-y-4 divide-y divide-gray-200 px-6 sm:max-w-2xl lg:max-w-4xl">
+        <div className="sm:flex sm:items-center">
+          <div className="mt-16 sm:flex-auto">
+            <h1 className="text-base font-medium text-gray-900">
+              Managed DataSources
+            </h1>
+
+            <p className="text-sm text-gray-500">
+              Fully managed and kept in sync in real-time with the products you
+              use.
+            </p>
+          </div>
+        </div>
+
+        <div className="mt-8 overflow-hidden">
+          <ul role="list" className="">
+            {managedDataSourcesLocal.map((ds) => {
+              return (
+                <li
+                  key={`managed-${ds.connectorProvider}`}
+                  className="px-2 py-4"
+                >
+                  <div className="flex items-center justify-between">
+                    <div className="flex items-center">
+                      {ds.enabled ? (
+                        <Link
+                          href={`/w/${
+                            owner.sId
+                          }/ds/${`managed-${ds.connectorProvider}`}`}
+                          className="block"
+                        >
+                          <p
+                            className={classNames(
+                              "truncate text-base font-bold",
+                              ds.enabled ? "text-violet-600" : "text-slate-400"
+                            )}
+                          >
+                            {ds.name}
+                          </p>
+                        </Link>
+                      ) : (
+                        <p
+                          className={classNames(
+                            "truncate text-base font-bold",
+                            ds.enabled ? "text-violet-600" : "text-slate-400"
+                          )}
+                        >
+                          {ds.name}
+                        </p>
+                      )}
+                    </div>
+                    <div>
+                      {!ds.enabled ? (
+                        <Button
+                          disabled={
+                            !ds.isBuilt ||
+                            isLoadingByProvider[ds.connectorProvider]
+                          }
+                          onClick={
+                            canUseManagedDataSources
+                              ? () => {
+                                  handleEnableManagedDataSource(
+                                    ds.connectorProvider as ConnectorProvider
+                                  );
+                                }
+                              : () => {
+                                  logger.info(
```

Comment:
cc @spolu If you click on "Get early access" we simply log it. Is that good enough ?

Reply:
as per below a small window,alert to invite to ping us